ayekan/prometheus/values.yaml

```diff
@@ -219,3 +219,11 @@ grafana:
           jsonData:
             handleGrafanaManagedAlerts: false
             implementation: prometheus
+
+# these components are not present in the Rancher clusters
+kubeScheduler:
```

Comment:
I'm not sure I'm following... All 3 of these processes are running on the ayekan nodes.